node_modules/oae-core/activity/js/activity.js

```diff
@@ -122,8 +122,9 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
             infinityScroll = $('.oae-list', $rootel).infiniteScroll(url, {
                 'limit': 10
             }, '#activity-items-template', {
-                'postProcessor': processActivities,
-                'emptyListProcessor': handleEmptyResultList
+                'ariaLive': 'off',  // Notifications will alert screen readers
```

Comment:
Maybe we should set `'ariaLive': 'assertive'` to the element containing the title of the activity that was added?